javaagent-extension-api/src/main/java/io/opentelemetry/javaagent/extension/instrumentation/InstrumentationModule.java

```diff
@@ -145,7 +145,7 @@ public boolean isHelperClass(String className) {
    * advices, grouped by {@link ClassRef#getClassName()}.
    *
    * <p>The actual implementation of this method is generated automatically during compilation by
-   * the {@code io.opentelemetry.javaagent.tooling.muzzle.collector.MuzzleCodeGenerationPlugin}
+   * the {@code io.opentelemetry.javaagent.muzzle.generation.collector.MuzzleCodeGenerationPlugin}
    * ByteBuddy plugin.
```

Comment:
WDYT about 
```suggestion
   * the {@code io.opentelemetry.instrumentation.javaagent-codegen} gradle plugin.
```
instead? It might be a better idea to point to the gradle plugin instead of one of its internal classes.